pkg/services/authn/clients/password.go

```diff
@@ -51,9 +49,8 @@ func (c *Password) AuthenticatePassword(ctx context.Context, r *authn.Request, u
 	for _, pwClient := range c.clients {
 		identity, clientErr := pwClient.AuthenticatePassword(ctx, r, username, password)
 		clientErrs = multierror.Append(clientErrs, clientErr)
-		// for invalid password or if the identity is not found by a client continue to next one
+		// we always try next client on any error
 		if clientErr != nil {
-			c.log.Warn("failed to authenticate password identity", "client", pwClient, "error", clientErr)
```

Comment:
nit: could perhaps leave this log in a debug level?

Reply:
Yea that can be useful